modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -78,6 +77,15 @@ function MultipleChoice(props) {
     Array.of(existingAnswer[1].value).flat()
     // Only the internal values are stored, turn them into pairs of [label, value] by using their displayedValue
     .map((item, index) => [Array.of(existingAnswer[1].displayedValue).flat()[index], item]);
+  // When opening a form, if there is no existingAnswer but there are AnswerOptions specified as default values,
+  // display those options as selected and ensure they get saved unless modified by the user, by adding them to initialSelection
+  if (initialSelection.length == 0 && defaultAnswers.length > 0) {
+    initialSelection = defaultAnswers.map(item => [item[LABEL_POS], item[VALUE_POS]]);
+    // If there are more default values than the specified maxAnswers, only take into account the first maxAnswers default values.
+    if (maxAnswers > 0 && initialSelection.length > maxAnswers) {
+      initialSelection = initialSelection.slice(0, maxAnswers);
+    }
+  }
```

Comment:
This block + line 64 could be replaced with the following (not tested):

```suggestion
  if (initialSelection.length == 0) {
    initialSelection = defaults.filter(item => item[IS_DEFAULT_ANSWER_POS])
      // If there are more default values than the specified maxAnswers, only take into account the first maxAnswers default values.
      .slice(0, maxAnswers || defaults.length)
      .map(item => [item[LABEL_POS], item[VALUE_POS]]);
  }
```